README.md

```diff
@@ -4,48 +4,57 @@ The purpose of this repo is to build, test and run omnibuses.
 
 ## Omnibus
 
-The main purpose of omnibus is to prepare the EVM script that will be executed if the vote is successful.
-During the run, omnibus will call newVote function of the Voting contract with the prepared script and description.
-Voting EVM script will be built from items that are defined in the omnibus.
+The main purpose of the omnibus is to prepare the EVM script that will be executed if the vote is successful. The voting EVM script is built from the omnibus items defined in the omnibus. During the run, the omnibus will call the
+```
+newVote(bytes executionScript, string metadata, bool ???, bool ???)
+```
+function of the voting contract with the prepared script and description.
+
+## Omnibus Item
+
+Each omnibus is made up of items. An omnibus item is the basic building block of each omnibus. It represents a single on-chain action (such as changing protocol settings, granting or revoking access, transferring tokens, etc). In code, it is represented as an object containing
+* `title` - arbitrary name of the current item
+* `evmCall` - ????
+* `expectedEvents` - expected on-chain events that should be fired after the vote is executed (used as a means of self-verification procedure)
+
+```typescript
+interface OmnibusItem {
+  title: string;
+  evmCall: FormattedEvmCall;
+  expectedEvents: EventCheck[];
+}
+```
 
 ## Keystores
 
-Keystores allow you to store your private keys in a secure way and use them in the omnibus run process.
+Keystores allow you to securely store your private keys and use them in the process of running omnibuses.
 
 ## Installation
 
 1. Clone the repo
 2. Install dependencies via `pnpm install`
-3. types will be generated automatically via postinstall script
+3. Types will be generated automatically via postinstall script
 4. Seed the `.env` file from the `.env.example`
 
 ## Writing omnibuses
 
 You need to create a new file in the [omnibuses](./src/omnibuses) folder.
 Naming convention is to name omnibuses `${YYYY_MM_DD}.ts`.
 
-### Omnibus Item
-
-Each omnibus consists of items.
-Omnibus item is an object that contains item title, EVM script call,
-and expected events that should be fired after the vote is executed.
+Writing an omnibus essentially means packing a bunch of omnibus items into an `omnibuses.create` call along with the additional parametres:
+* `network` - ???
```

Comment:
@Ivan-Feofanov please help with two fields desc here

Reply:
network should be one of allowed network names. For now it's "mainnet" and "goerly"